code/modules/jobs/job_types/adventurer/types/combat/cleric.dm

```diff
@@ -18,63 +18,54 @@
 	..()
 	H.virginity = TRUE
 
-	armor = /obj/item/clothing/armor/cuirass // Halfplate has been made heavy armor, billions must make due.
-	shirt = /obj/item/clothing/shirt/shortshirt/random
+	armor = /obj/item/clothing/armor/cuirass/iron // Adventurers don't get steel anymore
+	shirt = /obj/item/clothing/armor/gambeson
+	neck = /obj/item/clothing/neck/chaincoif/iron
+	head = /obj/item/clothing/head/helmet/ironpot
 	pants = /obj/item/clothing/pants/trou/leather
 	shoes = /obj/item/clothing/shoes/boots/leather
+	gloves = /obj/item/clothing/gloves/leather
 	belt = /obj/item/storage/belt/leather
 	beltl = /obj/item/weapon/mace
 	beltr = /obj/item/storage/belt/pouch/coins/poor
+	backpack_contents = list(/obj/item/storage/keyring/priest = 1)
```

Comment:
why ? they do not belong to the church , plus this key pack gives them access to the priest room and the inquisition which they should absolutely have no access being an adventurer